example/integrations/tensorflow/Dockerfile.cpu

```diff
@@ -9,5 +9,5 @@ RUN  apt-get update --fix-missing \
 && apt-get install -y git \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
-RUN pip install tf-nightly-gpu \
+RUN pip install tf-nightly \
```

Comment:
if we're not using this dockerfile  in the yaml file, please move this to different director.

Reply:
hmm, this is a dockerfile that can actually work with the latest master branch.

Reply:
if `volcanosh/example-tf:0.0.1` is not built from this dockerfile, please move it out and give a new name to `volcanosh/example-tf:0.0.1`.

Reply:
New images now:
1. example-tf-cpu: 0.0.1
2. example-tf-gpu: 0.0.1

Reply:
As discussed before, this dockerfile is an example; and `example-tf:xxx` is retag from tensorflow-benchmark. So we need to make this message clear to others.

Reply:
`example-tf:xxx` is only used by OpenLab now, and those can be removed in the future, we don't use `example-tf:xxx` image

Reply:
hm.. I'm confused about current status :( please make sure dockerfile & image in yaml align, all the time.

Reply:
@k82cn , ha let me summarize the history.

1. We used to have `example-tf.yaml` and `dockerfile`, in which `example-tf:0.0.1` image is used. But now they are all have been removed.
2. `dockerfile.cpu` is used to generate` example-tf-cpu.0.0.1` image, which is been used in `tf-example-cpu.yaml`
3. there is another `dockerfile.gpu` is used to generate` example-tf-gpu.0.0.1` image, and which is been used in `tf-example-gpu.yaml`

4. openlab generated another image example-tf:0.0.3 which is used for compare test with kubeflow.

Reply:
> openlab generated another image example-tf:0.0.3 which is used for compare test with kubeflow.

do not use the same name of image; that's a really long history ...

Reply:
> > openlab generated another image example-tf:0.0.3 which is used for compare test with kubeflow.
> 
> do not use the same name of image; that's a really long history ...

Will be renamed when this cycle of tests finished.